server/repository/feed/store.js

```diff
@@ -1,28 +1,41 @@
 'use strict';
 
-module.exports = new (class extends Map {
-  addContext(user, context) {
-    const record = this._findOrCreate(user);
-    record.contexts.push(context);
-  }
+const Promise = require('bluebird');
+const store = require('../../shared/store');
 
-  removeContext(user, filterFn) {
-    const record = this.get(user.id);
-    if (!record) return;
-    record.contexts = record.contexts.filter(it => !filterFn(it));
-    if (record.contexts.length <= 0) this.delete(user.id);
-  }
+const getKey = id => `active-user-${id}`;
 
-  toJSON() {
-    return Array.from(this.entries())
-      .reduce((acc, [key, value]) => ({ ...acc, [key]: value }), {});
-  }
+async function addContext(user, context) {
+  const key = getKey(user.id);
+  const record = await findOrCreate(user);
+  const contexts = [...record.contexts, context];
+  return store.set(key, { ...record, contexts }, 0);
```

Comment:
Would aliasing `0` with something like `NO_EXPIRATION_TTL` / `INFINITE_TTL` make it easy to infer what's going on?